check_http_json.rb

```diff
@@ -18,23 +18,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-
 # Requires.
-require 'rubygems'
 require 'json'
 require 'net/http'
 require 'net/https'
 require 'uri'
 require 'optparse'
 require 'timeout'
 
+# Manage Nagios messages and exit code
+module Nagios
```

Comment:
The module is accessible everywhere, but so is a global variable, for example. I realise that this isn't The Ruby Way™, but it _is_ simple, and that matters. Can you see a way to implement this functionality using variable instead of a module? If not, that's cool, but I'd like to explore that option.